recipes/igrafx-mining-sdk/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "igrafx-mining-sdk" %}
+{% set version = "2.32.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/igrafx_mining_sdk-{{ version }}.tar.gz
+  sha256: 63ae24a229eafdf9910412a374fee32f64df785ff05449af09db6c14796bd654
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.10.0,<3.12.1
+    - pip
+    - poetry ==1.7.1
+  host:
+    - pip
+    - python >=3.10.0,<3.12.1
+    - poetry ==1.7.1
+    - SQLAlchemy >=2.0.29
+  run:
+    - python >=3.10.0,<3.12.1
+    - poetry ==1.7.1
+    - requests >=2.30.0
+    - pydruid >=0.6.5
+    - pandas >=2.0.0,<=2.2.1
+    - SQLAlchemy >=2.0.29
+    - networkx >=3.0.0,<3.3.0
+    - numpy >=1.25.0
+    - toml ==0.10.2
+    - python-dotenv ==1.0.1
+    - pytest ==8.1.1
+    - pytest-dependency >=0.5.1
+
+test:
+  source_files:
+    - igrafx_mining_sdk/
+    - tests/
+  imports:
+    - igrafx_mining_sdk
+    - pytest
+  commands:
+    - pytest ./tests/test_01_column_mapping.py
```

```suggestion
    - pytest ./tests/test_01_column_mapping.py
    - pip check
```
To check dependencies